EpiAware/docs/src/showcase/replications/mishra-2020/index.jl

```diff
@@ -50,6 +47,29 @@ This tutorial introduces the basic functionality of `EpiAware`. `EpiAware` is a
 It is common to conceptualise the generative process of public health data, e.g a time series of reported cases of an infectious pathogen, in a modular way. For example, it is common to abstract the underlying latent infection process away from downstream issues of observation, or to treat quanitites such as the time-varying reproduction number as being itself generated as a random process.
 
 `EpiAware` is built using the [`DynamicPPL`](https://github.com/TuringLang/DynamicPPL.jl) probabilistic programming domain-specific language, which is part of the [`Turing`](https://turinglang.org/dev/docs/using-turing/guide/) PPL. The structural concept behind `EpiAware` is that each module of an epidemiological model is a self-contained `Turing` [`Model`](https://turinglang.org/DynamicPPL.jl/stable/api/#DynamicPPL.Model-Tuple{}); that is each module is an object that can be conditioned on observable data and sampled from. A complete `EpiAware` model is the composition of these objects using the [`@submodel`](https://turinglang.org/DynamicPPL.jl/stable/api/#DynamicPPL.@submodel) macro.
+
+## Pluto code blocks
+Note that `Pluto.jl` notebooks operate on single lines of code or code blocks encapsulated by `let ... end` and `begin ... end`. The reason is that `Pluto` notebooks are reactive, and re-run downstream code after changes with downstreaming determined by a tree of dependent code blocks. If you are running this example from the REPL, you can either use the code blocks and/or run each line individually.
+
+_NB:_ The difference between `let ... end` blocks and `begin ... end` blocks are that the `let ... end` type of code block only adds the final output/return value of the block to scope, like an anonymous function, whereas `begin ... end` executes each line and adds defined variables to scope.
+
+## Environment for this notebook and running it from the REPL
+We are going to use the `docs/` Julia environment for this notebook which is set by the `Project.toml` file in the `EpiAware/docs` folder, and check-out the latest version of `EpiAware` directly from its source code using `Pkg.develop`.
+
+From the context of this notebook we can find these _relative_ to this notebook using the `@__DIR__` macro, which returns the path of this notebook. If you are running this from the REPL from the root of the `RtWithoutRenewal` repository then replace the following:
+
+```julia
+docs_dir = \"EpiAware/docs\"
+pkg_dir = \"EpiAware\"
+```
+"
+
+# ╔═╡ 27d73202-a93e-4471-ab50-d59345304a0b
+md"
+## Dependencies for this notebook
+In the code block above we activated the environment we wanted, this determines the exact versions of the extra packages we want to use as dependencies. We also ran `Pkg.instantiate` to make sure we have locally downloaded these packages.
+
```

Comment:
Ok-- so by setting `pkg_dir = \"EpiAware\"` this is the equivalent in python of installing requirements or in R as using a renv file to install dependencies, but we still need to import/load them downstream

Reply:
No not really. This is all local dev stuff (I.e in the same way as in R) not what you would do with an actual package.

Reply:
@SamuelBrand1 I really don't think we need any environment stuff here and can just ask users to install via using  prompts when now they can see the dependencies that this has.